stock_indicators/_cslib/__init__.py

```diff
@@ -2,7 +2,7 @@
 Skender.Stock.Indicators
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-This module loads `Skender.Stock.Indicators.dll`, which is a compiled library package
+This module loads `Skender.Stock.Indicators.dll`(v1.22.4), which is a compiled library package
 from <https://github.com/DaveSkender/Stock.Indicators>, written in C#.
 
 It is currently using `.NET Standard 2.0`.
```

Comment:
Does Python.NET 3.x support newer .NET versions that are 2-3x faster than .NET Standard?

Reply:
Yeah, not now, but it will support .NET 6.0 in pythonnet v3.0.

Reply:
.NET Standard 2.1 is supported with current pythonnet version. Is it better than 2.0, right?

Reply:
Yah, I’d use the newest possible, they’re progressively faster, especially when you get to .NET Core versions.

Reply:
Okay, then I'll change the dll to .NET Standard 2.1.

Reply:
Yes, .NET Standard is a "bridge" standard between .NET Framework (older) and .NET Core (newer); it has the broadest compatibility between those two worlds.  Each progressive version is measurably faster.

Reply:
Ah, thank you for letting me know :)